frontend/packages/dev-console/locales/es/devconsole.json

```diff
@@ -490,7 +490,7 @@
   "Git repo URL": "URL del repositorio Git",
   "Create Sample application": "Crear aplicación de muestra",
   "The Devfile at {{filePath}} is recommended.": "Se recomienda el Devfile en {{filePath}}.",
-  "The func.yaml file is recommended.": "Se recomienda el archivo func.yaml.",
+  "The Serverless function strategy is recommended.": "The Serverless function strategy is recommended.",
```

Comment:
Missing translation

Reply:
This is okay since ES translations are not part of this batch.